### DISCUSSION TITLE
Feature request: Support AES decryption

### DISCUSSION THREAD

**Discussion by KurtMar** [cthackers/adm-zip#491] [Ideas]
In reference to https://github.com/cthackers/adm-zip/issues/259#issuecomment-793456717 and https://github.com/cthackers/adm-zip/issues/467#issuecomment-2002733865 this is a feature request for AES support, so that users are fully aware it is not currently supported and can track if it ever becomes supported.

**Comment by 5saviahv** [comment]
There is two methods for AES encryption PKWARE and WinZip method.

Even when AES decryption by PKWARE method is possible to add - all documentation is available in their APPNOTE. There is Copyright clause on this. Everyone who wish to implement it, have to get their written permission.

WinZip method what was created when PKWARE kept his method secret is open but sometimes documentation is lacking there.

**Comment by cthackers** [comment]
Also, there should be no dependencies added to the library. So any implementation needs to be done here.
